Makefile

```diff
@@ -61,11 +62,27 @@ container-uninstall:
 	sudo hack/update_binaries.sh delete
 
 .PHONY: godep
+ifeq ($(GO111MODULE),on)
+godep:
+	$(GO_BIN) build -v ./...
```

Comment:
this is just the dep step, building shouldn't happen in here?

Reply:
fixed!